src/Domain/Entity/Promotion.php

```diff
@@ -61,6 +61,11 @@ public function getSynopses(): Synopses
         return $this->synopses;
     }
 
+    public function getShortSynopsis(): string
```

Comment:
If a promotion already has a`getSynopses()` method, why do we need this one?

Reply:
Utility - it all cases i can think of so far we want the short Synopsis and dont need the others. To save a bit of indirection it's a bit neater to do this (same as things like $service->isTV() are aliases to $service->getNetwork()->isTv()). CoreEntity also provides a getSynopsis and a getShortSynopsis

Reply:
But the `isTv()` utility is there because Network might be null....  
I feel it's unnecessary. But the code looks fine if you still feel it's useful.